manager/api_driver.py

```diff
@@ -1,21 +1,27 @@
+import logging
 from abc import ABC, abstractmethod
-from typing import override
+from typing_extensions import override
```

Comment:
I added these, since `from typing import override` wasn't found

Reply:
I think that is a python version issue, have you ran it under the dev `nix-shell` (or with python 3.12)